accessibility-in-action/platform
# feat: display sign language videos in content (Resolves #1268)

Resolves #1268 

Adds a component and model for displaying Sign Language videos along side site content. Placeholders have been added to a few pages, including the home page and the about individual page.

To test:

- Login and enable the sign language translations setting from the web accessibility page
- Visit the home page and the about individual page to create the models
- Modify the database by adding in the video URLs. (you could also run the Interpretation factory and set the relevant, route, name, and namespaces).

```json
{
	"ase": "https://vimeo.com/766454375",
	"fcs": "https://vimeo.com/766455246"
}
```

- go back to the home page and about individual page
- click on the "sign language" buttons to see the video displayed.
- if you change between French and English you'll get different videos.

### Prerequisites

If this PR changes PHP code or dependencies:

- [x] I've run `composer format` and fixed any code formatting issues.
- [x] I've run `composer analyze` and addressed any static analysis issues.
- [x] I've run `php artisan test` and ensured that all tests pass.
- [x] I've run `composer localize` to update localization source files and committed any changes.

If this PR changes CSS or JavaScript code or dependencies:

- [x] I've run `npm run lint` and fixed any linting issues.
- [x] I've run `npm run build` and ensured that CSS and JavaScript assets can be compiled.
